recipes/zeromq/zeromq.inc

```diff
@@ -0,0 +1,21 @@
+inherit autotools pkgconfig c++ auto-package-utils auto-package-libs
+
+RECIPE_TYPES = "machine sdk"
```

Comment:
you're right, I need more coffee. It might be that we did not know for sure where the curve_gen tool should be executed. I would prefer we live the SDK build since it works and is being used.